src/component/Inputcity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import Time from './Time'
+import Weatherrr from './Weatherrr'
+const weather = require('openweather-apis');
```

Comment:
can't you use import here? 
```suggestion
import weather from 'openweather-apis';
```